src/atom.cpp

```diff
@@ -2294,6 +2294,11 @@ void *Atom::extract(char *name)
   if (strcmp(name,"dpdTheta") == 0) return (void *) dpdTheta;
   if (strcmp(name,"edpd_temp") == 0) return (void *) edpd_temp;
 
+  // when used with fix property/atom
```

Comment:
why is this needed? why not call `atom->find_custom()` directly?

Reply:
I now have removed all changes from `atom.cpp` to `library.cpp`